Orm/Xtensive.Orm.Tests/Issues/IssueGitHub0070_IncludeProviderParameterMismatchWithinParameterContext.cs

```diff
@@ -31,41 +31,39 @@ public class IssueGitHub0070_IncludeProviderParameterMismatchWithinParameterCont
     protected override DomainConfiguration BuildConfiguration()
     {
       var config = base.BuildConfiguration();
-      config.Types.Register(Assembly.GetExecutingAssembly(), typeof (Token).Namespace);
+      config.Types.Register(typeof(Token).Assembly, typeof(Token).Namespace);
       return config;
     }
 
     [Test]
     public void TwoDifferentInStatementsInTheSameQuery()
     {
-      using (var session = Domain.OpenSession()) {
-        using (session.OpenTransaction()) {
-          var tokenId = Guid.NewGuid();
-          var token = new Token(session, tokenId) {
-            Salt = -1
-          };
-          session.SaveChanges();
-          var query = session.Query.All<Token>()
-            .Where(t => t.Salt.In(-1, 1) && t.Id.In(tokenId, Guid.Empty));
-          Assert.AreSame(token, query.FirstOrDefault());
-        }
+      using (var session = Domain.OpenSession())
+      using (session.OpenTransaction()) {
+        var tokenId = Guid.NewGuid();
+        var token = new Token(session, tokenId) {
+          Salt = -1
+        };
+        session.SaveChanges();
+        var query = session.Query.All<Token>()
+          .Where(t => t.Salt.In(-1, 1) && t.Id.In(tokenId, Guid.Empty));
+        Assert.AreSame(token, query.FirstOrDefault());
       }
     }
 
     [Test]
     public void TwoSimilarInStatementsInTheSameQuery()
```

Comment:
This test case doesn't show anything different than the first one. Maybe we should just remove it.

Reply:
I would keep the one with different types of parameters it shows the problem better and clearer. I guess you meant the same